terraform/helm/genesis/files/genesis.sh

```diff
@@ -23,6 +23,25 @@ LARGER_STAKE_AMOUNT=${LARGER_STAKE_AMOUNT:-1}
 # TODO: Fix the usage of this below when not set
 RANDOM_SEED=${RANDOM_SEED:-$RANDOM}
 
+ENABLE_MULTICLUSTER_DOMAIN_SUFFIX=${ENABLE_MULTICLUSTER_DOMAIN_SUFFIX:-false}
+MULTICLUSTER_DOMAIN_SUFFIXES_DEFAULT="forge-multiregion-1,forge-multiregion-2,forge-multiregion-3"
+MULTICLUSTER_DOMAIN_SUFFIXES_STRING=${MULTICLUSTER_DOMAIN_SUFFIXES_STRING:-$MULTICLUSTER_DOMAIN_SUFFIXES_DEFAULT}
```

Comment:
Try to wrap references with curly braces everywhere, e.g. ${MULTICLUSTER_DOMAIN_SUFFIXES_DEFAULT}.